src/ActionModal/components/DelegationModal.vue

```diff
@@ -237,14 +237,14 @@ export default {
           type: transaction.REDELEGATE,
           validatorSourceAddress: this.from,
           validatorDestinationAddress: this.targetValidator.operatorAddress,
-          amount: uatoms(this.amount),
+          amount: this.amount,
```

Comment:
this is actually needed. the transaction data needs the actual amount and not the display amount (so actual uatoms)